doc/lsst.obs.decam/index.rst

```diff
@@ -4,30 +4,48 @@
 lsst.obs.decam
 ###############
 
-.. Paragraph that describes what this Python module does and links to related modules and frameworks.
+This package is an interface between Dark Energy Camera (DECam) images and the Rubin/LSST Science Pipelines. It primarily supports processing of raw images,
+not DECam Community Pipeline "instcal" products.
 
-Write description here
+.. _lsst.obs.decam-using:
+
+Using lsst.obs.decam
+====================
+
+.. toctree::
+   :maxdepth: 1
+
+   using-obs-decam
```

Comment:
Does anything in restructured text do anything? 🙃 I copied lots of choices that were made in the `lsst.verify` docs, since Jonathan generally knows what he is doing. I promise to actually build the docs and make sure they look OK.

Reply:
Ok, this did put linked sections into this table of contents.